solr/solrj/src/test/org/apache/solr/client/solrj/TestLBHttp2SolrClient.java

```diff
@@ -205,16 +202,31 @@ public void testTwoServers() throws Exception {
     }
   }
 
-  public void testReliability() throws Exception {
+  @LogLevel("org.apache.solr.client.solrj.impl.LBSolrClient=DEBUG")
```

Comment:
is this needed to make the test pass, or from working on the code?  the @logLevel?

Reply:
ah, I think I see why you have it from looking lower down in the LogReliablityTestSetup